ui.go

```diff
@@ -1171,6 +1171,15 @@ func (ui *ui) readNormalEvent(ev tcell.Event, nav *nav) expr {
 				ui.menuBuf = nil
 				return draw
 			}
```

Comment:
I'd check that val is not `nil` and `val[0]=='<'` around here, just in case.

Reply:
Done, thanks.